consensus/src/consensus/mod.rs

```diff
@@ -171,17 +173,16 @@ pub struct Consensus {
     pub counters: Arc<ProcessingCounters>,
 
     // Config
-    config: Config,
+    config: Arc<Config>,
```

Comment:
Why you made it an Arc?

Reply:
Because this struct is going to grow a lot and is shared by many sub-systems